css/style.css

```diff
@@ -0,0 +1,302 @@
+@font-face {
+  font-family: "PT Sans";
+  font-style: normal;
+  font-weight: 400;
+  src: url("../fonts/ptsans-400.woff2") format(woff2);
+  font-display: swap;
+}
+
+@font-face {
+  font-family: "PT Sans";
+  font-style: normal;
+  src: url("../fonts/ptsans-700.woff2") format(woff2);
```

Comment:
```suggestion
  src: url("../fonts/ptsans-400.woff2") format("woff2");
  font-display: swap;
}

@font-face {
  font-family: "PT Sans";
  font-style: normal;
  src: url("../fonts/ptsans-700.woff2") format("woff2");
```
Пропущены кавычки в наименовании формата. В результате шрифт не рендерится, вместо него Arial:
![image](https://user-images.githubusercontent.com/17790771/137263861-12f5d08a-d1fb-4543-99ce-4da2ccef9f55.png)